recipes/itk/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "4.11.0" %}
+
+package:
+  name: itk
+  version: "{{version}}"
+
+source:
+  url: https://github.com/InsightSoftwareConsortium/ITK/archive/v{{version}}.tar.gz
+  sha256: a1857aa89f02efce3e091d102fbff7b2c53fe40c7f74580a47236bfd846c4e8b
+
+build:
+    number: 0
+    skip: True   # [win and py36]
+    features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - toolchain
+    - cmake    >=3.3
+    - ninja
+    - expat    2.1.*
+    - hdf5     1.8.*
+    - jpeg     9*
+    - libtiff  4.0.*
+    - libpng   >=1.6.27,<1.7
+    - zlib     1.2.*
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
+  run:
+    - expat    2.1.*
+    - hdf5     1.8.*
+    - jpeg     9*
+    - libtiff  4.0.*
+    - libpng   >=1.6.27,<1.7
+    - zlib     1.2.*
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
+
+test:
+  files:
+    - example
+  commands:
+    - test -d $PREFIX/include/ITK*                        # [unix]
+    - if not exist %LIBRARY_INC%\\ITK* exit 1             # [win]
```

Comment:
Agreed this is not needed, with the exists tests.

The CMake example configuration mainly checks the proper existence of CMake files referencing library files, it does not check for headers.

Also running "itkTestDriver", does a run time check some ITK libraries too.